### ISSUE TITLE
Setup issues on Windows

### ISSUE THREAD

**Issue by muthu0101** [modess/git-pretty-stats#31]
I have attached 2 screen shot (Index page and next page), I am getting error on router page please guide me.

![1](https://cloud.githubusercontent.com/assets/5661907/6592452/bb9a6696-c7f4-11e4-8514-4dfb5ea71ff3.png)

![error](https://cloud.githubusercontent.com/assets/5661907/6592422/52ab70ee-c7f4-11e4-872a-991f50fd429c.png)

**Comment by modess** [comment]
Unfortunately the tool has not been tested properly in Windows environments, mainly because I develop on osx. It seems that it has issues dealing with file paths in Windows. 

I'm hoping someone will take a look at it, or I have to set up a Windows environment myself and fix it.

**Comment by muthu0101** [comment]
If you give basic guidance of the your code and how it works, I can contribute this fix.

**Comment by modess** [comment]
The issues should now have been resolved, I've tested it with XAMPP on Windows 7.

The problems were actually not from this repository, but in https://github.com/klaussilveira/gitter which it uses for git interactions. 

* The git binary path was not escaped properly to run in the shell, https://github.com/klaussilveira/gitter/pull/43
* Getting top level names didn't use the proper directory separator constant, https://github.com/klaussilveira/gitter/pull/44

I've included both in my fork and use that as a dependency for now. So in order to get everything up and running:
* Pull the latest changes
* Run `composer update`

Let me know if it works for you.
